src/components/ModalToolbar.tsx

```diff
@@ -0,0 +1,88 @@
+import { IonButton, IonHeader, IonIcon, IonText, IonToolbar } from '@ionic/react';
+import { closeOutline } from 'ionicons/icons';
+import React, { useEffect, useState } from 'react';
+import { Pages } from '../Models/Enums';
+import './ModalToolbar.css';
+
+interface ModalToolbarProps {
+    pageCallback: CallableFunction;
+    pageRendered: Pages;
+}
+
+const ModalToolbar: React.FC<ModalToolbarProps> = (props: ModalToolbarProps) => {
+    const formatTime = (originalTime: string) => {
+        const regex = /([0-9]+):([0-9]+):([0-9]+) ([A-Z]+)/;
```

Comment:
Seems like we're doing the time calculation twice. It may be better to keep all time functions in a .ts class with static methods